src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -892,6 +892,13 @@ private async Task<ContainerServiceManagedClusterResource> ProvisionManagedClust
                 };
 
                 cluster.PublicNetworkAccess = ContainerServicePublicNetworkAccess.Disabled;
+
```

Comment:
more concise (aka less distracting, for me at least): `(cluster.NetworkProfile ??= new()).OutboundType = ContainerServiceOutboundType.UserDefinedRouting;`

Reply:
Almost as concise, less esoteric:

> cluster.NetworkProfile ??= new()
> cluster.NetworkProfile.OutboundType = ContainerServiceOutboundType.UserDefinedRouting;